tests/integration/blackbox_test.py

```diff
@@ -206,11 +206,7 @@ def prop_assert(dr_resp, actual, expected):
         "inputs": [(i,) for i in range(100)],
         "invariants": {
             DRProp.cost: 14,
-            DRProp.lastLog: {
-                # since execution REJECTS for 0, expect last log for this case to be None
-                (i,): Encoder.hex(i * i) if i else None
-                for i in range(100)
-            },
```

Comment:
An [upgrade to go-algorand's dry run](https://github.com/algorand/go-algorand/pull/3929) is returning logs even when rejecting.